odnoklassniki-android-sdk-example/src/main/java/ru/ok/android/sdk/example/MainActivity.java

```diff
@@ -119,6 +99,43 @@ public void onError(String error) {
         });
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        mLoginBtn.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(final View view) {
+                mOdnoklassniki.requestAuthorization(MainActivity.this, REDIRECT_URL, false, OkScope.VALUABLE_ACCESS);
+            }
+        });
+
+        mOdnoklassniki.setOkListener(this);
```

Comment:
It is not required for this sample, thus might be misleading. I think it should have been removed.